Source/Microsoft.Teams.Apps.FAQPlusPlus/Cards/CardHelper.cs

```diff
@@ -87,15 +87,33 @@ public static string GetTicketDisplayStatusForSme(TicketEntity ticket)
         }
 
         /// <summary>
-        /// Returns a string that will display the given date and time in the user's local time zone, when placed in an adaptive card.
+        /// Returns a string that will display the given date and time in the user's local time zone in a thumbnail card.
         /// </summary>
         /// <param name="dateTime">The date and time to format.</param>
         /// <param name="userLocalTime">The sender's local time, as determined by the local timestamp of the activity.</param>
         /// <returns>A datetime string.</returns>
         public static string GetFormattedDateInUserTimeZone(DateTime dateTime, DateTimeOffset? userLocalTime)
         {
-            // Adaptive card on mobile has a bug where it does not support DATE and TIME, so for now we convert the date and time manually.
-            return dateTime.Add(userLocalTime?.Offset ?? TimeSpan.FromMinutes(0)).ToShortDateString();
+            // Keeping date format for ar-sa as invariant, since we don't want to convert the dates to islamic calendar dates.
```

Comment:
Is this truly a requirement? It feels wrong to be hardcoding ar/ar-sa, since that's not the only culture that uses a different calendar. th-th uses a ThaiBuddhistCalendar, and fa-ir uses PersianCalendar.

Reply:
Hello @aosolis,
Yes, We had a discussion with team and came to know we are not suppose to show date according Islamic calendar.
And right now we are supporting only Arabic from cultures which have different calendar, so added condition for AR only.